.github/config-schema.json

```diff
@@ -1022,6 +1022,29 @@
         }
       ]
     },
+    "mise": {
+      "default": {
+        "detect_extensions": [],
+        "detect_files": [
+          ".mise.toml",
```

Comment:
mise also supports `.config/mise.toml`, among others: https://mise.jdx.dev/configuration.html

Note in particular environment-specifig config files, which may be of particular interest here.